### ISSUE TITLE
Getting error when use addParticipant error : addParticipants: incoming change missing version id

### ISSUE THREAD

**Issue by supernocan** [pedroslopez/whatsapp-web.js#2067]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

when i try to add participant to group alyays error

Error: Evaluation failed: Error: addParticipants: incoming change missing version id
    at Object.<anonymous> (https://web.whatsapp.com/app.39064ec0dec361638b30.js:46:316531)
    at Generator.next (<anonymous>)
    at t (https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66483)
    at s (https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66694)
    at https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66753
    at Y (https://web.whatsapp.com/app.39064ec0dec361638b30.js:5:140102)
    at new y (https://web.whatsapp.com/app.39064ec0dec361638b30.js:5:132669)
    at Object.<anonymous> (https://web.whatsapp.com/vendor1~app.0c8e960e5dfd7020f562.js:2:66634)
    at Object.m (https://web.whatsapp.com/app.39064ec0dec361638b30.js:46:317336)
    at Object.t.addParticipants (https://web.whatsapp.com/app.39064ec0dec361638b30.js:46:314045)
    at ExecutionContext._evaluateInternal (C:\Users\AMIER\Documents\PROJECT\docker\node_js\wa-bot\whatsapp-api-tutorial\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:221:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (C:\Users\AMIER\Documents\PROJECT\docker\node_js\wa-bot\whatsapp-api-tutorial\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:16)
    at async GroupChat.addParticipants (C:\Users\AMIER\Documents\PROJECT\docker\node_js\wa-bot\whatsapp-api-tutorial\node_modules\whatsapp-web.js\src\structures\GroupChat.js:62:16)

### Expected behavior

Success to add contact to group

### Steps to Reproduce the Bug or Issue

1. Find Group By Name
2. Find Contact By Name
3. Add Contact to Group
4. Error

### Relevant Code

  const group_id = req.body.group_id;
  const contact = req.body.contact;
  client.getChats().then((chats) => {
    const myGroupName = 'TEST-BUAT-GROUP';
    const myGroup = chats.find((chat) => chat.name === myGroupName);
    client.getContacts().then((contacts) => {
        const contactToAdd = contacts.find(
            // Finding the contact Id using the contact's name
            (contact) => contact.name === 'Amier XL'
          );
        
        if (contactToAdd) {
            console.log('Sukses');
            myGroup.addParticipants([contactToAdd.id._serialized]);
        }else{
            console.log('Gagal');
        }
    });
})

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

No, I am not using Multi Device

### Environment

OS : Windows 11
Phone OS : Android
Whatsapp-web.js version :1.19.4
Node.js : v18.14.1

**Comment by karankarakoti** [comment]
Got the same error

**Comment by anmolthedeveloper** [comment]
Getting the same error when trying to add participants to a group

**Comment by totallynotdavid** [comment]
Same error! I'm trying to use removeParticipants instead, still got this message.

**Comment by reimonlp** [comment]
Same error, also got to scan QR session again. Might be related to some changes on last whatsapp update

**Comment by diegofemello** [comment]
Same error here
![image](https://user-images.githubusercontent.com/8761655/226121804-e30cff61-024d-4fe1-904e-b305f564b7a7.png)

**Comment by Bestfastfire** [comment]
Same here

**Comment by saifkhan7865** [comment]
same error was anyone able to fix it?

**Comment by saifkhan7865** [comment]
https://github.com/pedroslopez/whatsapp-web.js/pull/2087

@PurpShell this was how I fixed it, is it possible for you to test it out and check if it is fine or not

**Comment by supernocan** [comment]
> > #2087
> > @PurpShell this was how I fixed it, is it possible for you to test it out and check if it is fine or not
> 
> It worked here

can you explain the example script thats successfully, cause i have update to version 1.19.5 still got the same error

**Comment by saifkhan7865** [comment]
Can you go to my branch and check, it will work

**Comment by viniciusjosedev** [comment]
> 

I got it thanks to your solution. How did you get there?

**Comment by viniciusjosedev** [comment]
> This is how i fixed it. node_modules/whatsapp-web.js/src/util/injected.js I replaced window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[1]; With window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[0];

I got it thanks to your solution. How did you get there?

**Comment by walumbe** [comment]
> 

You're welcome.  I checked for solutions of other developers on stack overflow

**Comment by transpostw2** [comment]
> This is how i fixed it. node_modules/whatsapp-web.js/src/util/injected.js I replaced window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[1]; With window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[0];

getting the following error for removeParticipants

```
D:\whatsapp-bot\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:221
            throw new Error('Evaluation failed: ' + helper_js_1.helper.getExceptionMessage(exceptionDetails));
                  ^

Error: Evaluation failed: H
    at ExecutionContext._evaluateInternal (D:\whatsapp-bot\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:221:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (D:\whatsapp-bot\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:16)
    at async GroupChat.addParticipants (D:\whatsapp-bot\node_modules\whatsapp-web.js\src\structures\GroupChat.js:62:16)

Node.js v18.12.1
```

**Comment by jackkelly-bayliss** [comment]
> > This is how i fixed it. node_modules/whatsapp-web.js/src/util/injected.js I replaced window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[1]; With window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[0];
> 
> getting the following error for removeParticipants
> 
> ```
> D:\whatsapp-bot\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:221
>             throw new Error('Evaluation failed: ' + helper_js_1.helper.getExceptionMessage(exceptionDetails));
>                   ^
> 
> Error: Evaluation failed: H
>     at ExecutionContext._evaluateInternal (D:\whatsapp-bot\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:221:19)
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
>     at async ExecutionContext.evaluate (D:\whatsapp-bot\node_modules\whatsapp-web.js\node_modules\puppeteer\lib\cjs\puppeteer\common\ExecutionContext.js:110:16)
>     at async GroupChat.addParticipants (D:\whatsapp-bot\node_modules\whatsapp-web.js\src\structures\GroupChat.js:62:16)
> 
> Node.js v18.12.1
> ```

Can confirm I'm receiving the same error

**Comment by Solaimanx** [comment]
> fixed version id number #2087

This one worked for me . Thanks

**Comment by victormolinaasencios** [comment]
> This is how i fixed it. node_modules/whatsapp-web.js/src/util/injected.js I replaced window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[1]; With window.Store.GroupParticipants = window.mR.findModule('promoteParticipants')[0];

esto lo soluciona, ¿cuándo lo actualizan en master?
